src/layout-grid/cell.js

```diff
@@ -12,9 +12,22 @@ import type {CellPropsT} from './types.js';
 
 // When used with Grid, Cell does not actually return anything.
 // All of the props are forwarded to a StyledCell inside of Grid.
-export default function Cell({align, children, order, skip, span}: CellPropsT) {
+export default function Cell({
+  align,
+  children,
+  order,
+  skip,
+  span,
+  gridUnit,
```

Comment:
I think `gridUnit` probably shouldn't be on `Cell` since you wouldn't ever want to change a single cell's unit? (It should be on `StyledCell` though so that it can calculate things internally).

Reply:
done 👍